internal/dag/accessors_test.go

```diff
@@ -75,10 +76,38 @@ func TestBuilderLookupService(t *testing.T) {
 		},
 	}
 
+	annotatedService := &v1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        "annotatedservice",
+			Namespace:   "default",
+			Annotations: map[string]string{"projectcontour.io/upstream-protocol.tls": "6441,6442,ssl"},
```

Comment:
is this intended to have 6442 here? none of the ports below are 6442 (tests still work I believe bc we look up w/ name and/or number)

Reply:
Yes, this is intentionally an extra value to ensure that extra values were ignored.